lib/swagger-client.js

```diff
@@ -1460,22 +1460,8 @@ Operation.prototype.encodeQueryParam = function(arg) {
   return encodeURIComponent(arg);
 };
 
-/**
- * TODO revisit, might not want to leave '/'
- **/
 Operation.prototype.encodePathParam = function(pathParam) {
-  var encParts, part, parts, i, len;
```

Comment:
@mohsen1, according to the [URI template spec](http://tools.ietf.org/html/rfc6570):

- slashes need to be encoded for `{foo}` (the only variant supported in Swagger 2.0), and
- slashes don't need to encoded for `{+foo}`.

So both can be supported easily and within the same URL once the spec gains support for level 2 URI templating.